include/swift/AST/DiagnosticsSema.def

```diff
@@ -1122,6 +1122,9 @@ ERROR(invalid_dynamic_member_lookup_type,none,
       "@dynamicMemberLookup attribute requires %0 to have a "
       "'subscript(dynamicMember:)' method that accepts either "
       "'ExpressibleByStringLiteral' or a keypath", (Type))
```

Comment:
Not your bug, but could we fix the spelling of this to “key path” as it’s spelled elsewhere in user-facing documentation?

Reply:
Good spot, I've fixed it!